setup/setup.go

```diff
@@ -179,9 +178,20 @@ func providerAlreadyPresent(providers []provider.Provider, provider provider.Pro
 	return false
 }
 
-func getAppConfigForInstallation(installation string) provider.AppConfig {
+func getAppConfigForInstallation(installation string, domains []string) provider.AppConfig {
 	return provider.AppConfig{
-		Name:                 fmt.Sprintf("dex-operator-%s", installation),
+		Name:                 key.GetDexOperatorName(installation),
 		SecretValidityMonths: 6,
+		IdentifierURI:        key.GetIdentifierURI(key.GetDexOperatorName(installation)),
+		RedirectURI:          getGithubRedirectURLs(domains),
+	}
+}
+
+// This returns a comma seperated list of callback URLs for github applications
+func getGithubRedirectURLs(domains []string) string {
```

Comment:
dex-operator can not create github apps itself (github api does not allow it). However, since it does not need elevated access because of that, we can use the authenticated app itself for access via dex. For this to work we need to add the redirect URLs for dex instances we want to manage to the authenticated app.